### ISSUE TITLE
Make Cloudflare Cache Clear Requests Asynchronous?

### ISSUE THREAD

**Issue by blogtutor** [blogtutor/blog-tutor-support#192]
Can we make the CF Cache Clear requests async?  Sergio says "it should not be much of a problem" LOL.

**Comment by ecotechie** [comment]
We can add `'blocking' => false` to wp_remote_post() as an argument and it will not wait for a reply...

https://developer.wordpress.org/reference/classes/WP_Http/request/

> 'blocking'
>
>(bool) Whether the calling code requires the result of the request. If set to false, the request will be sent to the remote server, and >processing returned to the calling code immediately, the caller will know if the request succeeded or failed, but will not receive >any response from the remote server. Default true.

**Comment by blogtutor** [comment]
This would be great, except it "will not receive any response from the remote server"... so we won't know what the CF API's response actually was, and if it actually cleared the cache.

But perhaps we don't need to know that all the time?  _Most of the time_ it seems to work just fine, and we've now got 2+ years of experience with this. Perhaps we do change it to non-blocking, but add a "debug" option in our plugin settings so we can re-enable it if we need to troubleshoot?

**Comment by ecotechie** [comment]
True, not sure it matters (as you mentioned). So the #alerts-cloudflare Slack channel would be for debugging and only show info from sites with this new setting enabled. I could work on that if you'd like.
